src/openrct2-ui/windows/MapGen.cpp

```diff
@@ -116,58 +116,58 @@ static constexpr const int32_t WH = 273;
 
```

Comment:
Seems like the constants above went unused (?) If so, delete them

Reply:
They're used in the `WINDOW_SHIM` macro, and elsewhere in the file to create the window.